source/fem-dem/ib_particles_dem.cc

```diff
@@ -372,31 +382,72 @@ IBParticlesDEM<dim>::calculate_pw_contact_force(
   double wall_restitution_coefficient =
     parameters->wall_restitution_coefficient;
 
+  struct DefaultDBL_MAX
+  {
+    double value = DBL_MAX;
+  };
+  struct DefaultUINT_MAX
+  {
+    int value = UINT_MAX;
+  };
```

Comment:
I would make it a tiny class instead with a default constructor instead of a struct with a value assigned. Also define it outside of the routine.

Reply:
I move the definition of the struct in the .h.I kept the struct instead of the class since it is in line with other similar use of struct in the code. Also, the value is used outside of the struct and, as such, is public.